docs/source/notebooks/power_analysis.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>refer to treatment_time rather than the hard-coded 70 please</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/277/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>